### DISCUSSION TITLE
File path on Vercel?

### DISCUSSION THREAD

**Discussion by aximili** [vercel/community#3779] [Help]
### Summary

I am a noob. I created a [T3 app](https://create.t3.gg/) having a SQLite DB, located at `prisma/db.sqlite`

I have the db path in the environment variable:
`DATABASE_URL="file:./db.sqlite"`

which **works locally**, but on Vercel I got this error:
_Error querying the database: unable to open database file: /var/task/node_modules/.prisma/client/./db.sqlite_

**What is the path to my file in Vercel?**
I have tried changing the path to every path I could think of, including /var/task/prisma/db.sqlite, but it could never find the file.

### Steps to Reproduce

1. Create a [T3 app](https://create.t3.gg/), include all options including Prisma
2. Change Prisma db provider to "sqlite"
3. Set the DATABASE_URL env to the path of the file
4. Make sure you can access the db file locally (eg. `prisma.example.create(...)`)
5. Push everything to Git including the db file
6. Have Vercel read from the Git, set up the `DATABASE_URL` env var, and deploy

**Comment by Jmg-21** [comment]
Encounter this same issue just now. do you have any way to  solve this

**Comment by yurifw** [comment]
trying to solve a similar issue as well
